style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: orangered;
+}
```

Comment:
- [ ] Kindly use the class name from your `h1` element in your index file as the selector to use to style the content of the page instead of the `h1` tag directly. This is also part of the project [requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md)